Run example is error

ElasticJob-Lite  
 

Change the code
```java
    private static void setEmailProperties(final JobConfiguration jobConfig) {
        //jobConfig.getProps().setProperty(EmailPropertiesConstants.HOST, "host");
        jobConfig.getProps().setProperty(EmailPropertiesConstants.PORT, "465");
        jobConfig.getProps().setProperty(EmailPropertiesConstants.USERNAME, "username");
        jobConfig.getProps().setProperty(EmailPropertiesConstants.PASSWORD, "password");
        jobConfig.getProps().setProperty(EmailPropertiesConstants.FROM, "from@xxx.xx");
        jobConfig.getProps().setProperty(EmailPropertiesConstants.TO, "to1@xxx.xx,to1@xxx.xx");
    }

```
**Run :**  org.apache.shardingsphere.elasticjob.lite.example.JavaMain

### Actual behavior
```
[ERROR] 2020-10-30 16:48:34,185 --main-- [org.apache.zookeeper.server.NIOServerCnxnFactory] Thread Thread[main,5,main] died 
java.lang.NullPointerException: null
	at java.util.Hashtable.put(Hashtable.java:460) ~[na:1.8.0_231]
	at org.apache.shardingsphere.elasticjob.error.handler.email.EmailJobErrorHandler.createSessionProperties(EmailJobErrorHandler.java:78) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.error.handler.email.EmailJobErrorHandler.init(EmailJobErrorHandler.java:68) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.infra.spi.ElasticJobServiceLoader.newTypedServiceInstance(ElasticJobServiceLoader.java:83) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.error.handler.JobErrorHandlerFactory.createHandler(JobErrorHandlerFactory.java:51) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:77) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:64) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.JobScheduler.<init>(JobScheduler.java:85) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.OneOffJobBootstrap.<init>(OneOffJobBootstrap.java:40) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.example.JavaMain.setUpOneOffJobWithEmail(JavaMain.java:130) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.example.JavaMain.main(JavaMain.java:76) ~[classes/:na]

```

--- a/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/schedule/JobScheduler.java
+++ b/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/schedule/JobScheduler.java
@@ -82,10 +82,10 @@ public final class JobScheduler {
         setUpFacade = new SetUpFacade(regCenter, jobConfig.getJobName(), jobListeners);
         schedulerFacade = new SchedulerFacade(regCenter, jobConfig.getJobName());
         jobFacade = new LiteJobFacade(regCenter, jobConfig.getJobName(), jobListeners, findTracingConfiguration(jobConfig).orElse(null));
+        validateJobProperties(jobConfig);
         jobExecutor = new ElasticJobExecutor(elasticJob, jobConfig, jobFacade);
         String jobClassName = JobClassNameProviderFactory.getProvider().getJobClassName(elasticJob);
         this.jobConfig = setUpFacade.setUpJobConfiguration(jobClassName, jobConfig);
-        validateJobProperties();
         setGuaranteeServiceForElasticJobListeners(regCenter, jobListeners);
         jobScheduleController = createJobScheduleController();
     }
@@ -97,9 +97,9 @@ public final class JobScheduler {
         setUpFacade = new SetUpFacade(regCenter, jobConfig.getJobName(), jobListeners);
         schedulerFacade = new SchedulerFacade(regCenter, jobConfig.getJobName());
         jobFacade = new LiteJobFacade(regCenter, jobConfig.getJobName(), jobListeners, findTracingConfiguration(jobConfig).orElse(null));
+        validateJobProperties(jobConfig);
         jobExecutor = new ElasticJobExecutor(elasticJobType, jobConfig, jobFacade);
         this.jobConfig = setUpFacade.setUpJobConfiguration(elasticJobType, jobConfig);
-        validateJobProperties();
         setGuaranteeServiceForElasticJobListeners(regCenter, jobListeners);
         jobScheduleController = createJobScheduleController();
     }
@@ -114,11 +114,11 @@ public final class JobScheduler {
         return jobConfig.getExtraConfigurations().stream().filter(each -> each instanceof TracingConfiguration).findFirst().map(extraConfig -> (TracingConfiguration<?>) extraConfig);
     }
     
-    private void validateJobProperties() {
-        validateJobErrorHandlerProperties();
+    private void validateJobProperties(final JobConfiguration jobConfig) {
+        validateJobErrorHandlerProperties(jobConfig);
     }
     
-    private void validateJobErrorHandlerProperties() {
+    private void validateJobErrorHandlerProperties(final JobConfiguration jobConfig) {
         if (null != jobConfig.getJobErrorHandlerType()) {
             ElasticJobServiceLoader.newTypedServiceInstance(JobErrorHandlerPropertiesValidator.class, jobConfig.getJobErrorHandlerType(), jobConfig.getProps())
                     .ifPresent(validator -> validator.validate(jobConfig.getProps()));

